src/client/user/components/Drawer/ControlPanel/index.tsx

```diff
@@ -221,13 +222,25 @@ export default function ControlPanel() {
     <>
       Your link is <span className={classes.activeText}>active</span> and
       publicly accessible
+      <Typography
+        variant="body2"
+        className={classes.regularText}
+      >
+        Share your link through <a href="https://postman.gov.sg" target="_blank">Postman</a>.
+      </Typography>
     </>
   )
 
   const stateTitleInactive = (
     <>
       Your link is <span className={classes.inactiveText}>inactive</span> and
       not publicly accessible
+      <Typography
+        variant="body2"
+        className={classes.regularText}
+      >
+        Share your link through <a href="https://postman.gov.sg" target="_blank">Postman</a>.
+      </Typography>
```

Comment:
no duplicate code, please. also, should this only be shown when the link is active?